guides/common/modules/ref_ports-and-firewall-requirements.adoc

```diff
@@ -96,6 +96,7 @@ This behavior can be turned off using `{foreman-installer} --foreman-proxy-dhcp-
 | 443 |  TCP | HTTPS | Connections to the Pulp server in the {SmartProxy}
 | 9090 | TCP | HTTPS | Connections to the proxy in the {SmartProxy}
 | 80 | TCP | HTTP | Downloading a bootdisk (Optional)
+| 8443 | TCP | HTTPS | Subscription Management Services and host registration
```

Comment:
AFAIK this is incorrect. Satellite doesn't talk to port 8443 on Capsules, but rather the other way around. Clients talk to port 8443 on Capsules and **that** is a reverse proxy to Satellite.

It should be in the table Client to {SmartProxy} that doesn't exist now.

Reply:
I had a bad feeling about this. 
Thank you @ekohl

Reply:
@ekohl, @melcorr Thanks for your review and feedback. After further investigation and feedback also from a support engineer, port 8443 is correctly listed in table .Ports for Client to {SmartProxy} Communication of the Installing {SmartProxy} guide. So this update may not be required. Thanks.